src/legacy/core_plugins/kibana/migrations/migrations.js

```diff
@@ -58,8 +58,42 @@ function migrateIndexPattern(doc) {
   doc.attributes.kibanaSavedObjectMeta.searchSourceJSON = JSON.stringify(searchSource);
 }
 
+function removeMatchAllQueryIfLanguageNotDefined(doc) {
+  const searchSourceJSON = get(doc, 'attributes.kibanaSavedObjectMeta.searchSourceJSON');
```

Comment:
is get<string> more correct in this case?

Reply:
@VladLasitsa it's not 😞 . In JavaScript we don't have generics